firestore/google/cloud/firestore_v1/client.py

```diff
@@ -115,12 +119,18 @@ def _firestore_api(self):
         if self._firestore_api_internal is None:
             # Use a custom channel.
             # We need this in order to set appropriate keepalive options.
+
             channel = firestore_grpc_transport.FirestoreGrpcTransport.create_channel(
                 self._target,
                 credentials=self._credentials,
                 options={"grpc.keepalive_time_ms": 30000}.items(),
             )
 
+            if self._emulator_host is not None:
+                channel = firestore_grpc_transport.firestore_pb2_grpc.grpc.insecure_channel(
+                    self._emulator_host
+                )
+
```

Comment:
The "main" version above (lines 123-127) should be in an `else:` clause for this `if`.